spec/fixtures/pdf_fill/21P-530V2/merge_fields.json

```diff
@@ -93,7 +94,5 @@
     "hasPreviouslyReceivedAllowance":"YES",
     "noPreviouslyReceivedAllowance":null,
     "hasAllowanceStatementOfTruth":null,
-    "noAllowanceStatementOfTruth":null,
-    "hasTransportation":"YES",
-    "noTransportation":null
```

Comment:
Do these need to be removed from `lib/pdf_fill/forms/va21p530v2.rb`?

Reply:
Only 'noTransportation' was removed; 'hasTransportation' was just moved up, and 'noAllowanceStatementOfTruth' had a comma removed. 'noTransportation' has been removed from `lib/pdf_fill/forms/va21p530v2.rb`

Reply:
oh! I see now. Thanks 😄